dotnet/package/common.csproj

```diff
@@ -100,4 +109,52 @@
       <PackagePath>data</PackagePath>
     </Content>
   </ItemGroup>
+
+  <Target Name="GenerateBARManifest" >
```

Comment:
I liked how the Android side spelled out `Build Asset Registry` instead of using the BAR acronym as much:

https://github.com/xamarin/xamarin-android/pull/5808/files#diff-89e1be508ae40cfbe438271b97c7e1db7578dfc8e945b1577a3d64116574ea50R151

Is it worth doing that in a couple places here? The `bar-manifests` directory seems OK, but maybe the MSBuild targets could spell out the full name?